src/Administration/Resources/app/administration/src/module/sw-sales-channel/snippet/de.json

```
@@ -105,6 +105,10 @@
       "labelInputSnippet": "Textbaustein-Set",
       "buttonDeleteDomain": "Domain löschen",
       "messageDeleteDomainError": "Die Domain \"{url}\" konnte nicht gelöscht werden, da sie mit bestehenden Produktexporten verknüpft ist.",
+      "foreignKeyDelete": {
+        "title": "Verkaufskanal kann nicht gelöscht werden",
+        "message": "Der Verkaufskanal kann nicht gelöscht werden, da noch Zuordnungen zu {assignment} bestehen."
```

```suggestion
        "message": "Dieser Verkaufskanal kann nicht gelöscht werden, da noch von {assignment} verwendet wird."
```
Sounds a bit more natural 🙂